pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -480,6 +480,17 @@ def test_to_datetime_unparseable_ignore(self):
         s = "Month 1, 1999"
         assert pd.to_datetime(s, errors="ignore") == s
 
+    def test_to_datetime_unparseable_raise(self):
+        # GH#10720
+        date = "Month 1, 1999"
+        expected_args = (
+            f"Unknown string format: %s {date}. "
```

Comment:
I believe that the error is here, can you remove the ```%s```?

Reply:
this looks likely

Reply:
`AssertionError: Pattern "Unknown string format: %s Month 1, 1999. You can coerce to NaT by passing errors='coerce'" not found in "Unknown string format. You can coerce to NaT by passing errors='coerce'"`
That is output from tests on Linux py36_minimum_versions & macOS py36_macos. There's no date at all. On other platforms test passes.